docs/liquidity_baking.md

```diff
@@ -0,0 +1,308 @@
+  _Parameters_:
+    - __to__: the account address that will receive the XTZ amount
+    - __tokensSold__: the amount of tzBTC to sell
+    - __minXtzBought__: the minimum amount of XTZ expected to be received
+    - __deadline__: the expiry time of the transaction
+- [__%xtzToToken__](https://gitlab.com/dexter2tz/dexter2tz/-/blob/liquidity_baking/dexter.liquidity_baking.mligo#261): exchange XTZ for tzBTC  
+  _Parameters_:
+    - __to__: the account address that will receive the tzBTC tokens
+    - __minTokensBought__: the minimum amount of tzBTC expected to be received
+    - __deadline__: the expiry time of the transaction
+- [__%tokenToToken__](https://gitlab.com/dexter2tz/dexter2tz/-/blob/liquidity_baking/dexter.liquidity_baking.mligo#L334): used as an intermediary to faciliate the exchange of tzBTC and XTZ between 2 contracts  
+  _Parameters_:
+    - __outputDexterContract__: the contract address to send the tokens to
+    - __minTokensBought__: the minimum amount of tzBTC tokens expected to be received
+    - __to__: the recipient of the tokens for the transaction sent to the `outputDexterContract` address
+    - __tokensSold__: the amount of tokens to be sold
+    - __deadline__: the expiry time of the transaction
+- [__%addLiquidity__](https://gitlab.com/dexter2tz/dexter2tz/-/blob/liquidity_baking/dexter.liquidity_baking.mligo#L188): provision of XTZ and tzBTC to the contract and minting of LP tokens  
+  _Parameters_:
+    - __owner__: the account address that will be credited with the LP tokens
+    - __minLqtMinted__: the minimum amount of LP tokens expected to be minted
+    - __maxTokensDeposited__: the maximum amount of tzBTC tokens expected to be withdrawn from the sender's balance
+    - __deadline__: the expiry time of the transaction
+- [__%removeLiquidity__](https://gitlab.com/dexter2tz/dexter2tz/-/blob/liquidity_baking/dexter.liquidity_baking.mligo#L220): burning of LP tokens and credit of XTZ and tzBTC  
+  _Parameters_:
+    - __to__: the account address that will be credited with XTZ and tzBTC tokens
+    - __lqtBurned__: the amount of LP tokens to burn
+    - __minXtzWithdrawn__: the minimum amount of XTZ expected to be credited
+    - __minTokensWithdrawn__: the minimum amount of tzBTC tokens expected to be credited
+    - __deadline__: the expiry time of the transaction
+
+### Interacting with the entrypoints with Taquito
+The main friction point of interacting with the LB contract in JavaScript is about simulating the calculations of the expected token outputs Michelson does.
+
+- __%tokenToXtz__:
+```ts
+import { TezosToolkit } from "@taquito/taquito"
+
+// to take into account the subsidy added to the LB contract
+// when the transaction will be baked
+const creditSubsidy = (xtzPool: BigNumber | number): BigNumber => {
+    if (BigNumber.isBigNumber(xtzPool)) {
+      return xtzPool.plus(new BigNumber(2500000));
```

Comment:
To clarify what the 2500000 refers to, consider defining a constant (ie: liquidityBakingSubsidy = 2500000)